graviton/README.md

```diff
@@ -1,9 +1,12 @@
 <!-- markdownlint-disable no-inline-html -->
 <!-- markdownlint-disable ol-prefix -->
+<!-- markdownlint-disable first-line-h1 -->
+<p align="center"><img  width=100%  src="https://infura-ipfs.io/ipfs/QmUJQFQET7VHyfDepM9CNhGBEabjH42XzK39kAisPWdE3K"  border="0" /></p>
 
-# GRAVITON (aka the TEAL Blackbox Toolkit): Program Reporting and Testing via Dry Runs
+Graviton (aka the TEAL Blackbox Toolkit): Program Reporting and Testing via Dry Runs
 
-<img width="345" alt="http://cds.cern.ch/record/2315186/files/scoap3-fulltext.pdf" src="https://user-images.githubusercontent.com/291133/160721859-21a3560a-0a82-4249-aa54-5ede4c60f8d2.png">
+<!-- Please let me know if you'd like to keep or you think the logo in the header is fine since it also encapsulates the diagram in the letter G -->
```

Comment:
@tzaffi , let me know if you'd like to keep the image here or you agree that removing it since it's also displayed on the logo looks better (in either case i'll make a further tweak and push the change removing this comment)

Reply:
Thank you for @tzaffi I merged you PR with the version bump and `thanks` markdown. Just this comment left, let me know if u are ok with me deleting the second image or you still want to keep it